index.Rmd

```diff
@@ -588,24 +594,24 @@ freq(a, value = -2)$count
 # but some elements of the list are impredictibly corrupted, so the list looks as follows
 b <- a
 c <- NA
-list <- list(a, b, c)
```

Comment:
Calling this variable "list" could create confusion for students, especially when the function "list()" is also being used a lot in this chunk of code. To show exemplary scripting habits, changed the variable name to something a bit more descriptive to facilitate interpretation.